app/src/main/java/edu/uc/hickmadc/customnotifications/ui/main/MainViewModel.kt

```diff
@@ -11,11 +11,13 @@ class MainViewModel : ViewModel() {
     }
 
     fun getAllGroups() : ArrayList<Group> {
-        var groups : ArrayList<Group> = ArrayList()
```

Comment:
groups variable value never changed so it is a constant. Also moved it to the top of the class in another commit https://kotlinlang.org/docs/coding-conventions.html#class-layout

Reply:
Good.